hot-reload-analysis/src/main/kotlin/org/jetbrains/compose/reload/analysis/MemberInfo.kt

```diff
@@ -16,7 +16,8 @@ data class FieldInfo internal constructor(
     val fieldId: FieldId,
     val isStatic: Boolean,
     val initialValue: Any?,
-) : MemberInfo() {
+    val resourceContentHash: Int?,
```

Comment:
I think we should call this something more generic. 
The idea here would be the following: 

Right now, we do have this additional hash that we calculate and re-use. Its coming from the resources content. If we want to include more information, instead of introducing more hash fields, we could just mix it into this one hash. 

How about 'extraHash' ?